packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -95,6 +95,11 @@ _The key elements of the component's public API surface:_
     -   topRight - ![Top Right Quadrant](./Resources/top_right.png)
     -   bottomRight - ![Bottom Right Quadrant](./Resources/bottom_right.png)
     -   `orientation` - represent the orientation of the notch on the wafer map outline (only visual). As only four static orientations are possible, it can be represented by an Enum with the following values: top, bottom, left, right.
+    -   `origin` - represents the origin coordinates of the dies grid for rendering the wafer map. Leaving the value `undefined` will set the value to the minimum corner of the bounding box of the dies coordinates. It will be an object of the type `PointCoordinates` with the following attributes:
+        -   x: number
+        -   y: number
```

Comment:
Could we have originX and originY properties?
Is there significant risk  / concern of the properties being set one at a time? I think with the batching implementation there really isn't a concern.

Reply:
Yeah, a nice benefit of that is that they could be attributes since they'd have primitive types, whereas an object can't be expressed as an attribute easily. Though that's not a huge deal since some of the existing configuration is already objects/arrays. But still better to try to limit that when possible.